misc/packaging/apache/ansible-runner-service.spec

```diff
@@ -66,7 +66,8 @@ install -m 644 ./misc/packaging/logrotate/ansible-runner-service %{buildroot}%{_
 install -m 644 ./ansible_runner_service.py %{buildroot}%{python3_sitelib}/runner_service
 
 %post
-[[ -f /var/log/ovirt-engine/ansible-runner-service.log ]] && chcon -t httpd_log_t /var/log/ovirt-engine/ansible-runner-service.log
+semanage fcontext -a -t httpd_log_t /var/log/ovirt-engine/ansible-runner-service.log 2> /dev/null || semanage fcontext -m -t httpd_log_t /var/log/ovirt-engine/ansible-runner-service.log
```

Comment:
Shouldn't we also check if /var/log/ovirt-engine/ansible-runner-service.log exists?

Reply:
This only adds the tag to the database or updates it if there was previously added.